tools/replay/replay_settings.h

```diff
@@ -106,6 +106,7 @@ static void PrintUsage(const char* exe_name)
     GFXRECON_WRITE_CONSOLE("          \t\tAvailable formats are:");
     GFXRECON_WRITE_CONSOLE("          \t\t    %s\t\tBitmap file format.  This is the default format.",
                            kScreenshotFormatBmp);
+    GFXRECON_WRITE_CONSOLE("          \t\t    %s\t\tPortable Network Graphics Format file format.", kScreenshotFormatPng);
```

Comment:
The "Format" came back?  "Portable Network Graphics Format file format"

Reply:
Ugh.  Will fix